app/assets/javascripts/components/dashboard/create_task.jsx

```diff
@@ -6,11 +6,43 @@ class TaskCreationForm extends DefaultModal {
 
   constructor(props) {
     super();
+    this.state = {
+    };
+    this.select = this.select.bind(this);
+    this.InputField = this.InputField.bind(this);
   }
 
-  print = () => {
-    console.log("The modal is closing")
-    this.closeModal();
+  InputField({ id, label, help, ...props }) {
+    const { Checkbox, Radio, FormGroup, ControlLabel, FormControl, Button, HelpBlock, TextArea } = ReactBootstrap;
+    return (
+      <FormGroup controlId={id} onChange={this.select}>
+        <ControlLabel>{label}</ControlLabel>
+        <br/>
+        <FormControl {...props} />
+        {help && <HelpBlock>{help}</HelpBlock>}
+      </FormGroup>
+    );
+  }
+
+  select = (event) => {
+    this.setState({[event.target.name]: event.target.value});
+  }
+
+  submit = () => {
+    let clientInput = parseInt(_client_input.getCleanedMentionedUsers()[0]);
+    let userInput = parseInt(_user_input.getCleanedMentionedUsers()[0]);
+    const payload = {
+      client_id: clientInput,
+      user_id: userInput,
+      description: this.state.description,
+      due_date: this.state.dueDate,
+      title: this.state.name
+    };
+    Requester.post('/api/tasks', payload).then((data) => {
+      this.closeModal();
+    }).catch((data) => {
+      // Error message
+    });
```

Comment:
for now you can do `console.error(data)` so you get a big annoying error box instead of letting it fail silently